src/modules/main/store/mainSlice.ts

```diff
@@ -58,8 +58,30 @@ export default mainSlice.reducer;
 
 export const fetchImportantPosts = (): AppThunkType => async (dispatch) => {
   try {
-    const posts = await Promise.resolve(MOCK_POSTS);
-    dispatch(loadImportant(posts));
+    const posts = await getPosts('important');
+    const loadedPosts = posts.data.content.map((post) => {
+      return {
+        author: {
+          photo: post.author.avatar,
```

Comment:
let's have the same property names  - `avatar` and `lastName`.
you can use https://lodash.com/docs/4.17.15#pick here.

e.g.
```
author: {
    ...pick(post.author, [
        avatar,
        firstName,
        lastName,
        id,
        mainInstitution
    ]),
}
```

Reply:
done